src/doc/concepts.ts

```diff
@@ -327,6 +339,10 @@ export const doc_concepts:Documentation = {
 		<code class="code">y=i//w</code>, which justifies the way
 		integer division and modulo are defined. By induction, this
 		argument extends to <i>n</i>-dimensional grids.
+        <br>
+        <center>
+        <img src="images/arrays.png"/>
```

Comment:
I cannot see the "images" folder, did you forget to `git add`? Also, including external images would disallow distributing the whole IDE as a single file. Please convert the images to data URIs with [a tool like this](https://onlinepngtools.com/convert-png-to-data-uri).